### PR TITLE
New footer mar 2019

### PR DESCRIPTION
**Pull request by Balex3000**
issue #30 

added links to our youtube and slack channels
slack icon is the old one. was not able to figure out why the new one was not retrieved.
at this point better old than nothing.
we'll keep the issue opened until the icon will be fixed
Tested in dev and test but on version before dockers. Was not able to spin dockers version.
Please test...

### CODE REVIEW

File: docker/templates/includes/_footer.html [hunk 1]
```diff
@@ -20,6 +20,19 @@
...
       </li>
+
+      <li class="list-inline-item">
+          <a href="https://www.youtube.com/channel/UChv_8AbbznYS-xa256nXV6w" target="_blank">
+            <i class="fab fa-youtube fa-2x"></i>
+          </a>
+      </li>
+		  
+      <li class="list-inline-item">
+          <a href="https://tinyurl.com/yan5yzp9" target="_blank">
```
Comment (lioramilbaum) [inline docker/templates/includes/_footer.html]: the endpoint should be: devopsloft.slack.com?

Comment (Balex3000) [reply docker/templates/includes/_footer.html]: this is the same tinyurl we are using for months.
it works for me:
when I use the url from Chrome where I usually use Slack, I get into #general directly
when I use it in other browser, I get into "Join the Slack workspace DevOps Loft" page

Comment (lioramilbaum) [reply docker/templates/includes/_footer.html]: lets change it devopsloft.slack.com

That should work. right?

Comment (ShironB) [reply docker/templates/includes/_footer.html]: I changed it manually to devopsloft.slack.com and it worked the same as the tinyurl.

In addition, just want to let you know that I faced the same issue with the CRLF line endings, had to change it to LF

Comment (lioramilbaum) [reply docker/templates/includes/_footer.html]: Than, maybe just add information on how to get into the Slack workspace.

@ShironB thanks for updating. @Balex3000 is working on the PR which should address this issue.

Comment (Balex3000) [reply docker/templates/includes/_footer.html]: changed the link to slack and retested on dev

**Comment by Balex3000** [comment]
I changed the link to slack and retested on dev

**Pull request merged**